### ISSUE TITLE
'#' comments are deprecated in PHP configuration files

### ISSUE THREAD

**Issue by ndarilek** [sandstorm-io/vagrant-spk#111]
Spun up a new LEMP stack today and got the following error in my app logs:

2016/01/04 17:03:01 [error] 19#0: *2 FastCGI sent in stderr: "PHP message: Fatal error: Comments starting with '#' are deprecated in Unknown on line 1" while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "ceca7e9216ba13f1d457915e864631ba.local.sandstorm.io:6080"

A bit of Googling led me here:

http://stackoverflow.com/questions/14074101/getting-comments-starting-with-are-deprecated-message-via-cli

which suggests that '#' comments in PHP configs are deprecated in favor of ';'.

Not sure whether this will work, but I've updated all of my shell scripts to use ; comments and will regenerate the VM. I'll report back on whether it fixes the issue, but thought I'd at least log the error here.

**Comment by paulproteus** [comment]
Thanks a bundle @ndarilek for filing this.

Looks like we should adjust
https://github.com/sandstorm-io/vagrant-spk/blob/master/stacks/lemp/setup.sh#L43
accordingly.

**Comment by ndarilek** [comment]
Updating all #s to ;s in setup.sh resolved this for me.

**Comment by paulproteus** [comment]
I've manually tested the fix now. For what it's worth, I can't actually
reproduce the problem.

@ndarilek what version of PHP are you using?

Maybe you can run

dpkg -l | grep php

and provide the output.

And/or if you can test the fix in #112 that'd be splendid.

**Comment by ndarilek** [comment]
Testing the PR may be a bit tough since I've already made the changes 
needed to work in my local scripts. However, here are my versions of the 
PHP package. If that doesn't reproduce the issue for you then I'll 
regenerate the package with #112 and see if that does it.

```
ii  dh-php5                        0.2 all          debhelper add-on to 
handle PHP PECL extensions
ii  php-pear                       5.6.14+dfsg-0+deb8u1 all          
PEAR - PHP Extension and Application Repository
ii  php5-cli                       5.6.14+dfsg-0+deb8u1 amd64        
command-line interpreter for the php5 scripting language
ii  php5-common                    5.6.14+dfsg-0+deb8u1 amd64        
Common files for packages built from the php5 source
ii  php5-curl                      5.6.14+dfsg-0+deb8u1 amd64        
CURL module for php5
ii  php5-dev                       5.6.14+dfsg-0+deb8u1 amd64        
Files for PHP5 module development
ii  php5-fpm                       5.6.14+dfsg-0+deb8u1 amd64        
server-side, HTML-embedded scripting language (FPM-CGI binary)
ii  php5-json                      1.3.6-1 amd64        JSON module for php5
ii  php5-readline                  5.6.14+dfsg-0+deb8u1 amd64        
Readline module for php5
ii  php5-sqlite                    5.6.14+dfsg-0+deb8u1 amd64        
SQLite module for php5
ii  pkg-php-tools                  1.28 all          various packaging 
tools and scripts for PHP packages
```
